src/components/file/examples/file.tsx

```diff
@@ -10,7 +10,11 @@ import { Component, h, State } from '@stencil/core';
 })
 export class FileExample {
     @State()
-    private value: FileInfo = { filename: 'letter.docx', id: 123 };
+    private value: FileInfo = {
+        filename: 'letter.docx',
+        id: 123,
+        href: 'https://leekspin.com/',
```

Comment:
It feels like this makes more sense if it points to a file…

```suggestion
    private value: FileInfo = {
        filename: 'cute-puppy.jpg',
        id: 123,
        href: 'https://fastly.picsum.photos/id/237/536/354.jpg',
```

Reply:
Well… ok… but couldn't you have at least used a cute cat then? 🙈

Reply:
(Noticed now that my link didn't work. I think it only works for embedding. I meant to link to [this picture](https://fastly.picsum.photos/id/237/3500/2095.jpg?hmac=y2n_cflHFKpQwLOL1SSCtVDqL8NmOnBzEW7LYKZ-z_o).)

<img src="https://picsum.photos/id/237/367/267">

Reply:
If you snooze you lose 😛